c++/src/sargs/PredicateLeaf.cc

```diff
@@ -510,6 +510,7 @@ namespace orc {
         break;
       }
       case PredicateDataType::STRING: {
+        ///TODO: check lowerBound and upperBound as well
```

Comment:
This looks like irrelevant to `timestamp`. If we need this, can we do it in a separate JIRA, please?

Reply:
I will fix it in a separate JIRA and patch. But here I want to leave a mark in case I forget.

Reply:
In that case, let use an IDed TODO later.
```
- TODO ...
- TODO(ORC-665) ...
```